nss-cohort-36/bangazon-api-imagination-station-api
# added rest framwork stuff to requirements.txt

# Description

Updated requirements.txt for REST dependencies 

## Type of change
Please delete options that are not relevant.
# Testing Instructions

1. Make sure REST requirements are in requirements.txt.
1. `pip install -r requirements.txt`


# Checklist:
